test/parallel/test-https-agent-secure-protocol.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
+var assert = require('assert');
+var common = require('../common');
+
+if (!common.hasCrypto) {
+  common.skip('missing crypto');
+  return;
+}
+
+var https = require('https');
+var fs = require('fs');
+
+var options = {
+  key: fs.readFileSync(common.fixturesDir + '/keys/agent1-key.pem'),
+  cert: fs.readFileSync(common.fixturesDir + '/keys/agent1-cert.pem'),
+  ca:  fs.readFileSync(common.fixturesDir + '/keys/ca1-cert.pem')
+};
+
+
+var server = https.Server(options, function(req, res) {
+  res.writeHead(200);
+  res.end('hello world\n');
+});
+
+server.listen(0, function() {
+  const port = this.address().port;
+  var globalAgent = https.globalAgent;
+  globalAgent.keepAlive = true;
+  https.get({
+    path: '/',
+    port: port,
+    ca: options.ca,
+    rejectUnauthorized: true,
+    servername: 'agent1',
+    secureProtocol: 'SSLv23_method'
+  }, function(res) {
+    res.resume();
+    globalAgent.once('free', function () {
```

Comment:
Style issue: can you drop the blank in `function ()`.  Same issue a few lines below.